common/nodejs-utils/src/package_manager.rs

```diff
@@ -0,0 +1,36 @@
+use std::fmt::{Display, Formatter};
+use std::path::PathBuf;
+
+#[derive(Debug, PartialEq, Copy, Clone)]
+pub enum PackageManager {
```

Comment:
Is there any way we could merge this with the `PackageManager` defined https://github.com/heroku/buildpacks-nodejs/blob/a8738e0a0750b76015f9051c732ec61413c410ca/common/nodejs-utils/src/package_json.rs#L48-L51? It'd be nice if we didn't have two.

Reply:
I looked a this a bit closer and i think these are two separate concepts:

- `heroku_nodejs_utils::package_json::PackageManager` - represents the deserialized `packageManager` field found in a `package.json` file.
- `heroku_nodejs_utils::package_manager::PackageManager` - represents a package manager within the Node.js ecosystem.

I don't think it's necessary for these two things to be merged nor for them to have different names since they exist in different namespaces.